examples/animation.rs

```diff
@@ -0,0 +1,154 @@
+use bevy::{
+    diagnostic::{FrameTimeDiagnosticsPlugin, LogDiagnosticsPlugin},
+    prelude::*,
+};
+use bevy_ecs_tilemap::map::{
+    Tilemap2dGridSize, Tilemap2dSize, Tilemap2dTextureSize, Tilemap2dTileSize, TilemapId,
+    TilemapTexture,
+};
+use bevy_ecs_tilemap::tiles::{AnimatedTile, Tile2dStorage, TileBundle, TilePos2d, TileTexture};
+use bevy_ecs_tilemap::{Tilemap2dPlugin, TilemapBundle};
+use rand::{thread_rng, Rng};
+
+mod helpers;
+
+fn create_background_tilemap(commands: &mut Commands, asset_server: &Res<AssetServer>) {
+    let texture_handle: Handle<Image> = asset_server.load("tiles.png");
+
+    let tilemap_size = Tilemap2dSize { x: 20, y: 20 };
+    let texture_size = Tilemap2dTextureSize { x: 96.0, y: 16.0 };
+    let tile_size = Tilemap2dTileSize { x: 16.0, y: 16.0 };
+    let grid_size = Tilemap2dGridSize { x: 16.0, y: 16.0 };
+
+    // To create a map we use the Tile2dStorage component.
+    // This component is a grid of tile entities and is used to help keep track of individual
+    // tiles in the world. If you have multiple layers of tiles you would have a Tilemap2dStorage
+    // component per layer.
+    let mut tile_storage = Tile2dStorage::empty(tilemap_size);
+
+    // Create a tilemap entity a little early
+    // We want this entity early because we need to tell each tile which tilemap entity
+    // it is associated with. This is done with the TilemapId component on each tile.
+    let tilemap_entity = commands.spawn().id();
```

Comment:
The connection between this line and the component above is not obvious when first reading.

I'd swap the order, and then explicitly connect them.